src/datasources/results/components/editors/query-results/QueryResultsEditor.test.tsx

```diff
@@ -173,12 +196,12 @@ describe('QueryResultsEditor', () => {
       })
     });
 
-    test('should render empty workspaces and partnumbers when cache is empty', async () => {
+    test('should render empty workspaces when cache is empty', async () => {
       cleanup();
 
       const emptyDatasource = {
         workspacesCache: Promise.resolve(new Map()),
-        partNumbersCache: Promise.resolve([]),
+        productCache: Promise.resolve({ products: [] }),
```

Comment:
I think we should refrain from referring to things as `cache`. The fact that things are cached is an implementation detail.